dependencies.yaml

```diff
@@ -0,0 +1,127 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: none
+    includes:
+      - build_python
+      - checks
+      - cudatoolkit
+      - py_version
+      - run
+      - test_cpp
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+      - test_cpp
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  checks:
+    output: none
+    includes:
+      - checks
+      - py_version
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - dask/label/dev
+  - conda-forge
+  - nvidia
+dependencies:
+  build_python:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - &cmake_ver cmake>=3.23.1,!=3.25.0
+          - librmm==23.4.*
+      - output_types: conda
+        packages:
+          - fmt>=9.1.0,<10
+          - &gtest gtest==1.10.0.*
+          - &gmock gmock==1.10.0.*
+          - spdlog>=1.11.0,<1.12
+          - cython>=0.29,<0.30
+          - numpy>=1.21
```

Comment:
Should we use an anchor here and re-use this pinning version in the other places numpy is referenced? See `&gtest` / `*gtest` for an example.